transmogrifier/sources/xml/ead.py

```diff
@@ -31,7 +31,6 @@ def get_optional_fields(self, source_record: Tag) -> dict | None:
 
         # <archdesc> and <did> elements are required when deriving optional fields
         collection_description = self._get_collection_description(source_record)
-        collection_description_did = self._get_collection_description_did(source_record)
```

Comment:
Methods have been created for all fields that retrieved data from `<archdesc><did>`, thus this line of code is not needed!